Reference/conformance/conformance.pb.swift

```diff
@@ -163,7 +163,13 @@ struct Conformance_ConformanceRequest: SwiftProtobuf.Message {
   /// other serializer methods are defined in the SwiftProtobuf library. See the
   /// `Message` and `Message+*Additions` files.
   func traverse<V: SwiftProtobuf.Visitor>(visitor: inout V) throws {
-    try self.payload?.traverse(visitor: &visitor)
+    switch self.payload {
```

Comment:
I prefer this one. "if let x ... switch x" is an anti-pattern as far as I'm concerned, since the pattern matching in Swift is capable of resolving the optionality at the same time as the cases themselves.